[2024-03-03T03:42:05.833+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_6 scheduled__2016-12-19T00:00:00+00:00 [queued]>
[2024-03-03T03:42:05.845+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_6 scheduled__2016-12-19T00:00:00+00:00 [queued]>
[2024-03-03T03:42:05.847+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:42:05.867+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_6> on 2016-12-19 00:00:00+00:00
[2024-03-03T03:42:05.876+0000] {standard_task_runner.py:60} INFO - Started process 4691 to run task
[2024-03-03T03:42:05.881+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_6', 'scheduled__2016-12-19T00:00:00+00:00', '--job-id', '2143', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpw87zkpk0']
[2024-03-03T03:42:05.886+0000] {standard_task_runner.py:88} INFO - Job 2143: Subtask get_files_6
[2024-03-03T03:42:05.915+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:42:05.971+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_6 scheduled__2016-12-19T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:42:06.104+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_6' AIRFLOW_CTX_EXECUTION_DATE='2016-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-12-19T00:00:00+00:00'
[2024-03-03T03:42:08.762+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:42:08.782+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_6, execution_date=20161219T000000, start_date=20240303T034205, end_date=20240303T034208
[2024-03-03T03:42:08.825+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:42:08.857+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
